swift/ql/test/library-tests/dataflow/flowsources/webview.swift

```diff
@@ -23,13 +24,20 @@ class TestMessageHandler: WKScriptMessageHandler {
 func testJsContext(context: JSContext) {
     context.globalObject // SOURCE
     context.objectForKeyedSubscript("") // SOURCE
-    context.setObject(Exposed.self, forKeyedSubscript: "exposed" as! NSCopying & NSObjectProtocol)
 }
 
-class Exposed {
+protocol Exported : JSExport {
+    var tainted: Any { get }
+    func tainted(arg1: Any, arg2: Any)
+}
+class ExportedImpl : Exported {
     var tainted: Any { get { return "" } } // SOURCE
```

Comment:
We don't appear to catch this one.

Reply:
Good catch, there as an issue in how the source was modeled around fields. Fixed in b62ede1, thanks!